### PR TITLE
Update Cloudspades

### PR DESCRIPTION
**Pull request by pdimens**
While cloudspades builds to completion, there is an error when using the version from the conda recipe:
```
===== K21 started.


== Running: 
/home/pdimens/Documents/harpy/.snakemake/conda/f09094cb9c3ca647a3fd0548e0f7c996_/bin/sp
ades-core /home/pdimens/Documents/harpy/Assembly/spades/K21/configs/config.info 
/home/pdimens/Documents/harpy/Assembly/spades/K21/configs/isolate_mode.info

/home/pdimens/Documents/harpy/Assembly/spades/input_dataset.yaml:7:11: error: unknown 
enumerated scalar
  "type": "clouds10x"
          ^~~~~~~~~~~
Failed to load file 
"/home/pdimens/Documents/harpy/Assembly/spades/input_dataset.yaml"terminate called 
without an active exception


== Error ==  system call for: 
"['/home/pdimens/Documents/harpy/.snakemake/conda/f09094cb9c3ca647a3fd0548e0f7c996_/bin
/spades-core', '/home/pdimens/Documents/harpy/Assembly/spades/K21/configs/config.info',
'/home/pdimens/Documents/harpy/Assembly/spades/K21/configs/isolate_mode.info']" 
finished abnormally, OS return value: -6
None
```

However, this error does not appear if I create a new (and empty) conda environment, download the same tarball onto my system and run the compile script manually from within this conda env, without flags, and simply move the `bin` and `share` folders to the proper `${PREFIX}` locations. This PR is an attempt to replicate that with the conda recipe. 

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

### General instructions

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

### Instructions for avoiding API, ABI, and CLI breakage issues
Conda is able to record and lock (a.k.a. pin) dependency versions used at build time of other recipes.
This way, one can avoid that expectations of a downstream recipe with regards to API, ABI, or CLI are violated by later changes in the recipe.
If not already present in the meta.yaml, make sure to specify `run_exports` (see [here](https://bioconda.github.io/contributor/linting.html#missing-run-exports) for the rationale and comprehensive explanation).
Add a `run_exports` section like this:

```yaml
build:
  run_exports:
    - ...

```

with `...` being one of:

| Case                             | run_exports statement                                               |
| -------------------------------- | ------------------------------------------------------------------- |
| semantic versioning              | `{{ pin_subpackage("myrecipe", max_pin="x") }}`     |
| semantic versioning (0.x.x)      | `{{ pin_subpackage("myrecipe", max_pin="x.x") }}`   |
| known breakage in minor versions | `{{ pin_subpackage("myrecipe", max_pin="x.x") }}` (in such a case, please add a note that shortly mentions your evidence for that) |
| known breakage in patch versions | `{{ pin_subpackage("myrecipe", max_pin="x.x.x") }}` (in such a case, please add a note that shortly mentions your evidence for that) |
| calendar versioning              | `{{ pin_subpackage("myrecipe", max_pin=None) }}`    |

while replacing `"myrecipe"` with either `name` if a `name|lower` variable is defined in your recipe or with the lowercase name of the package in quotes.

### Bot commands for PR management

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

Note that the <code>@BiocondaBot please merge</code> command is now depreciated. Please just squash and merge instead.

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by pdimens** [comment]
@BiocondaBot please add label

**Comment by mencian** [comment]
LGTM, thanks @pdimens!

**Review by mencian — APPROVED** [review]

File: recipes/cloudspades/meta.yaml [hunk 1]
```diff
@@ -50,7 +50,5 @@ about:
   doc_url: "https://github.com/ablab/spades/tree/cloudspades-ismb"
 
 extra:
-  additional-platforms:
-    - linux-aarch64
```
Comment (martin-g) [inline recipes/cloudspades/meta.yaml:54]: What was the problem with aarch64 here ?

Comment (pdimens) [reply recipes/cloudspades/meta.yaml:54]: It fails to compile on ARM

Comment (martin-g) [reply recipes/cloudspades/meta.yaml:54]: I guess it failed to compile due to the usage of signed chars.
The support has been removed with this PR - https://github.com/bioconda/bioconda-recipes/pull/51456/files#diff-be671f9588fb1b742546ddcfd20773418719432b8e42799df8df52a9e7a3df9eL8 (`-fsigned-char` is gone)

What is the correct way to pass CXXFLAGS for this recipe ?

Comment (pdimens) [reply recipes/cloudspades/meta.yaml:54]: It seems to work correctly without any flags but the previous commits that borrowed the CXX flags from the regular `spades` recipe built, however the resulting installation had errors when running (same for the current `spades` recipe, which remains unresolved). AFAIK, there isn't an example of a proper spades ARM build on bioconda and the existing `spades` (regular) build is currently bugged and unresolved.

Comment (pdimens) [reply recipes/cloudspades/meta.yaml:54]: I know just about nothing about compilation, so I'm able to identify that there is an issue, but powerless to implement meaningful fixes.

**Pull request merged**